src/main/java/press/lis/greeb/Bot.java

```diff
@@ -32,7 +31,8 @@ public static void main(String[] args) {
     }
 
     public void onUpdateReceived(final Update update) {
-        if (update.hasMessage() && update.getMessage().hasText()) {
+        if (update.hasMessage() && update.getMessage().hasText() &&
+        update.getMessage().getText().equals("/subscribe")) {
```

Comment:
As far as I know, there could be another way of the message design called inline keyboard, the prototype of it as well will be useful:
https://core.telegram.org/bots/2-0-intro#new-inline-keyboards

I think we can use it in common chat to change the decision after the initial voting.